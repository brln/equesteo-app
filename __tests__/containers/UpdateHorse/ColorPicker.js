import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import ColorPickerContainer from '../../../containers/UpdateHorse/ColorPicker'


const mockStore = configureStore([thunk])

test('renders', () => {
  const horseID1 = 'horseID1'
  const store = mockStore(fromJS({
    pouchRecords: {
      horses: {
        horseID1: {
          _id: horseID1
        }
      }
    }
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <ColorPickerContainer
        horseID={horseID1}
      />
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
});
