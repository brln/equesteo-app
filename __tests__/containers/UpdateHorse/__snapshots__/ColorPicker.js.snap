// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": "100%",
      "padding": 20,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    />
    <View>
      <RCTSlider
        disabled={false}
        enabled={true}
        maximumValue={1}
        minimumValue={0}
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onSlidingComplete={null}
        onStartShouldSetResponder={[Function]}
        onValueChange={[Function]}
        step={0}
        style={
          Object {
            "height": 40,
          }
        }
        value={0.6127659574468085}
      />
      <RCTSlider
        disabled={false}
        enabled={true}
        maximumValue={1}
        minimumValue={0}
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onSlidingComplete={null}
        onStartShouldSetResponder={[Function]}
        onValueChange={[Function]}
        step={0}
        style={
          Object {
            "height": 40,
          }
        }
        value={0.9215686274509803}
      />
    </View>
  </View>
</View>
`;
